common/src/main/resources/tachyon-default.properties

```diff
@@ -94,20 +92,20 @@ tachyon.worker.network.netty.file.transfer=MAPPED
 tachyon.worker.memory.size=128MB
 tachyon.worker.network.netty.watermark.high=32KB
 tachyon.worker.network.netty.watermark.low=8KB
-tachyon.worker.allocate.strategy.class=tachyon.worker.block.allocator.MaxFreeAllocator
-tachyon.worker.evict.strategy.class=tachyon.worker.block.evictor.LRUEvictor
-tachyon.worker.evict.strategy.lrfu.step.factor=0.25
-tachyon.worker.evict.strategy.lrfu.attenuation.factor=2.0
+tachyon.worker.allocator.class=tachyon.worker.block.allocator.MaxFreeAllocator
+tachyon.worker.evictor.class=tachyon.worker.block.evictor.LRUEvictor
+tachyon.worker.evictor.lrfu.step.factor=0.25
+tachyon.worker.evictor.lrfu.attenuation.factor=2.0
```

Comment:
How do you want to organize the entries? Shall we sort them alphabetically to minimize the potential merge conflicts?